src/lib/init.js

```diff
@@ -59,10 +59,7 @@ export function init(configUpdates) {
 				function digitrustPixel() {
 					getVendorConsents([metadata.digitrustVendorId], (result) => {
 						if (result && result.vendorConsents && result.vendorConsents[metadata.digitrustVendorId]) {
-							let iframe = document.createElement('iframe');
-							iframe.src = config.digitrustIframeUrl;
-							iframe.style.width = "0px";
-							iframe.style.height = "0px";
+							let iframe = createIframe(config.digitrustIframeUrl, "0px", "0px");
 							document.body.appendChild(iframe);
```

Comment:
How about moving this call in the function? Also, I suggest chaning the function's name to 'createInvisibleIframe`.